index.html

```diff
@@ -15,15 +22,24 @@
 					<a href="#" class="logo">
 						<img src="./images/Welcome.svg" alt="Welcome" />
 					</a>
-					<a href="#" class="menu">
-						<img src="./images/menu.svg" alt="Drop down menu" />
-					</a>
+					<ul class="mobile-icons">
+						<li><i class="fa fa-bars fa-xl hamburger"></i></li>
+						<li><i class="fa fa-xmark fa-solid fa-xl close"></i></li>
+					</ul>
 				</div>
+				<!-- mobile menu -->
+				<ul class="options">
+					<li>
+						<a href="#portfolio" id="portfolio-link" class="option">Portfolio</a>
+					</li>
+					<li><a href="#about" id="about-link" class="option">About</a></li>
+					<li><a href="#contact" id="contact-link" class="option">Contact</a></li>
+				</ul>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 3-menu-mobile](https://user-images.githubusercontent.com/69638013/161971865-f7007043-d7f5-4d2e-aa3b-e826d40dd1be.png) |  ![aaaa](https://user-images.githubusercontent.com/69638013/161971260-5dab31c4-e694-4996-9d46-fd804bb7b371.png)



- [ ] It would be better for the background not to be scrollable when the mobile nav is open 👍🏿 

- Adding these functionalities will make your design good and look exactly like the Figma design, great job so far 💪🏿